cmd/update.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
+)
+
+var updateCmd = &cobra.Command{
+	Use: "update",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 2 {
+			// TODO: Improve this message
+			return fmt.Errorf("Invalid arguments, maximum 1 arguments possible")
+		}
+		return nil
+	},
+	Short: "update the component",
```

Comment:
It might be better to replace this with something like this: "Change the source of a component."

Reply:
@kadel Done